geoips/commandline/ancillary_info/test_data.py

```diff
@@ -1,24 +1,18 @@
-# # # This source code is protected under the license referenced at
-# # # https://github.com/NRLMMD-GEOIPS.
```

Comment:
Does this need to stay in?

Reply:
I think that's a question for @mindyls but probably, yes.

Reply:
Mindy says it isn't a huge deal so just gonna resolve and move on. It'll return automagically when its absence would be an issue.